### PR TITLE
Add option to configure kernel module parameters

### PR DESCRIPTION
**Pull request by SlothOfAnarchy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,15 +11,16 @@ An apt-based package manager with source version 16.04 or later, as ZFS isn't in
...
+| `zvols`                        |        `[]`         | List of zvols defined by a `name` and a dict of `attributes` (mandatory for each entry)                                                                                                           |
+| `zfs_default_enable_scrub`     |       `True`        | Whether to enable zpool scrubbing by default                                                                                                                                                      |
+| `zfs_scrub_frequency`          |      `weekly`       | Frequency of zfs scrubbing - already running scrubs will be aborted. See [systemd.time](https://www.freedesktop.org/software/systemd/man/systemd.time.html#Calendar%20Events) for allowed values. |
+| `zfs_kernel_module_parameters` |        `{}`         | Dict containing ZFS kernel module options to be set in `/etc/modprobe.d/zfs.conf`/.]                                                                                                              |
```
Comment (ghost) [inline README.md]: Please remove the ]

File: templates/zfs.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#{{ ansible_managed }}
```
Comment (ghost) [inline templates/zfs.conf.j2]: Can you add a blank between the hash and the comment?

File: templates/zfs.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+{% if zfs_kernel_module_parameters %}
+options zfs{% for key,value in zfs_kernel_module_parameters.iteritems() %} {{ key }}={{ value }}{% endfor %}
```
Comment (ghost) [inline templates/zfs.conf.j2]: Beautiful :)

**Review by ghost — APPROVED** [review]

**Review by ghost — DISMISSED** [review]
Wrong button (again)

**Comment by SlothOfAnarchy** [comment]
@dasJ please rereview.

**Review by dasJ — CHANGES_REQUESTED** [review]
There's still the `]`

**Review by dasJ — APPROVED** [review]

**Comment by SlothOfAnarchy** [comment]
Sorry, now it's gone.
Overlooked it and thought you meant the `/` I removed before...

**Pull request merged**